Rename Valid language tag to Known primary language tag (#1738)
act-rules/act-rules.github.io: _rules/html-page-lang-valid-bf051a.md

#### Failed Example 1

This `html` element has a `lang` attribute with a [known primary language tag][].

```html
<html lang="em-US"></html>
```

#### Failed Example 2

This `html` element has a `lang` attribute with a [known primary language tag][].

```html
<html lang="#1"></html>
```

#### Failed Example 3

> The `lang` attribute of this page is an [iso 639.2][] three letters code, which is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` attribute of this page is an [iso 639.2][] three letters code, which has no [known primary language tag][].
```

#### Failed Example 4

> The `lang` attribute of this page is a [grandfathered tag][grandfathered tags], which is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` attribute of this page is a [grandfathered tag][grandfathered tags], which has no [known primary language tag][].
```
